hack/setup.sh

```diff
@@ -46,6 +62,7 @@ main() {
                 case $command in
                 cluster)
                         start_registry
+                        start_repository
```

Comment:
I'm not sure I'd want a repository and gitea to be started everything I use this script to start a local cluster with cartographer.

I'd be in favor of making the gitea and gitops setup a separate step (and making the gitops testing separate as well) as they are a bit heavier to test and require more setup that we may not want to go through every time we are running tests locally. 

If not, at the very least I think we should defer this until `example-dependencies` so that the repository isn't setup unless you are setting up the examples. But as I said, I think I'd prefer to have the gitops setup and testing as a separate command. 

What do you think?

Reply:
I moved `start_repository` to `example_dependencies`. I've kept all the tests together. My expectation is that the examples are run by the end to end tests, and that devs aren't running that e2e on their local machines.